Package.swift

```diff
@@ -1,7 +1,15 @@
+// swift-tools-version:5.0
 import PackageDescription
 
 let package = Package(
     name: "Vinyl",
-    exclude: ["Carthage", "VinylTests"]
+    products: [
+        .library(name: "Vinyl", targets: ["Vinyl"]),
+    ],
+    targets: [
+        .target(name: "Vinyl", path: "Vinyl"),
+        .testTarget(name: "VinylTests", path: "VinylTests"),
```

Comment:
```suggestion
        .testTarget(name: "VinylTests", path: "VinylTests", dependencies: ["Vinyl"])),
```

The test target needs `Vinyl` as a dependency.